wps_tools/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+
+"""Top-level package for wps-tools."""
+
+from .__version__ import __author__, __email__, __version__  # noqa: F401
```

Comment:
This will probably be changed with the comment below.

Reply:
Since this is the top level, how do I properly import these variables from `setup.py` even when it's one directory above?

Reply:
What do you need to import these for?

Reply:
I was just going off what's done in `sandpiper`. Will this `__init__` file be empty, then?

Reply:
Ya we can just leave it empty!